resources/content/config-content/haproxy/content/etc/haproxy/haproxy.cfg.ftl

```diff
@@ -83,6 +83,7 @@ backend ${listener.uuid}_${backend.uuid}_backend
         
 </#list>
 </#list>
-<#else>
-listen web 0.0.0.0:9
 </#if>
+
+listen default 0.0.0.0:42
```

Comment:
You are right, it can conflict. I just wanted to reserve some port for LB health check, so user won't have to define health check for each LB service in his environment, and health check will be enabled for every LB service by default. And we can just document that port 42 is reserved for health checks.  I can re-work it the other way - let user specify the port/health check, and configure haproxy accordingly if you think that would be a more valid solution.